tests/communication/CMakeLists.txt

```diff
@@ -51,13 +82,43 @@ target_include_directories(app PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 # Target
 # ######################################################################################################################
 
+# Enable AESNI for posix 32bit
+target_compile_options(mbedTLS INTERFACE -mpclmul -msse2 -maes)
+
+set(aoscore_source_dir "${CMAKE_CURRENT_SOURCE_DIR}/../../../aos_core_lib_cpp")
```

Comment:
Move it to top and use in other places where aos_core_lib_cpp is referenced

Reply:
Done